org.eclipse.jdt.debug.tests/testfiles/Ambiguity/a/Sample.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
```

Comment:
when adding new files you may change it to your or employers name

Reply:
I can go with my employers name.